controllers/job.controller.js

```diff
@@ -0,0 +1,85 @@
+const Job = require('../models/job.model')
+const Company = require('../models/company.model');
+const constants = require('../utils/constants');
+
+exports.createJob = async (req,res)=>{
+    try{
+        const jobData = {
+            title : req.body.title,
+            description : req.body.description,
+            vacancies : req.body.vacancies,
+            postedBy : req.user._id,
+            company : req.user.companyId
```

Comment:
In the case of the admin, we should explicitly pass the company id in the request body